src/index_runner/es_indexers/sample_set.py

```diff
@@ -28,13 +34,52 @@ def get_sample(sample_info):
         "version": "1.1"
     }
     resp = requests.post(url=config()['sample_service_url'], headers=headers, data=json.dumps(payload))
+    if not resp.ok:
+        raise RuntimeError(f"Returned from sample service with status {resp.status_code} - {resp.text}")
     resp_json = resp.json()
     if resp_json.get('error'):
         raise RuntimeError(f"Error from SampleService - {resp_json['error']}")
     sample = resp_json['result'][0]
     return sample
 
 
+def _flatten_meta(meta, prefix=None):
+    """ Flattens metadata fields in a Sample object. Fields are concatenated into a 
```

Comment:
Tests failing due to flake8 complaining about trailing whitespace on this line. It's dumb i know